sdks/java/build-tools/beam-linkage-check.sh

```diff
@@ -88,29 +102,25 @@ function runLinkageCheck () {
     fi
 
     set +e
-    ./gradlew -Ppublishing -PjavaLinkageArtifactIds=$ARTIFACT ${BASELINE_OPTION} :checkJavaLinkage
+    set -x
+    ./gradlew -Ppublishing -PskipCheckerFramework -PjavaLinkageArtifactIds=$ARTIFACT_LIST ${BASELINE_OPTION} :checkJavaLinkage
     RESULT=$?
     set -e
+    set +x
     if [ "$MODE" = "validate" ]; then
       echo "`date`:" "Done: ${RESULT}"
       ACCUMULATED_RESULT=$((ACCUMULATED_RESULT | RESULT))
     fi
   done
 }
 
+echo "Establishing baseline linkage for $(git rev-parse --abbrev-ref $BASELINE_REF)"
+git -c advice.detachedHead=false checkout $BASELINE_REF
+runLinkageCheck baseline
```

Comment:
The only nontrivial shared code between `runLinkageCheck baseline` and `runLinkageCheck validate` is the loop over the artifact lists. So if we didn't need the loop, we may as well just inline the functions.